### ISSUE TITLE
the question about span loss?

### ISSUE THREAD

**Issue by zyxdSTU** [ShannonAI/mrc-for-flat-nested-ner#1]
Thanks for open source. 
```
if start_positions is not None and end_positions is not None:
            loss_fct = CrossEntropyLoss() 
        
            start_loss = loss_fct(start_logits.view(-1, 2), start_positions.view(-1))
            end_loss = loss_fct(end_logits.view(-1, 2), end_positions.view(-1))
            total_loss = start_loss + end_loss + span_loss
````
From code of above, I don't know how to understand the variable ‘’span_loss”.
Can someone help me?

**Comment by littlesulley** [comment]
hi, thanks for your focus on our work. 
For `span_loss`, you can refer to section 3.3 and section 3.4 for detail. In brief, the `span_loss` is used to judge whether a (start, end) pair can form an entity since there are inevitably some illegal entities considering all possible (start, end) pairs.

**Comment by zwkatgithub** [comment]
span_loss is not defined

**Comment by zyxdSTU** [comment]
> span_loss is not defined

Yes, You're a white man.  I can't find any place that define the variable "span_loss‘’， so I ask this question, but they explain the concept.So I want to know someone can find the implement of
the 'span_loss'.

**Comment by zwkatgithub** [comment]
I am trying to 

> > span_loss is not defined
> 
> Yes, You're a white man. I can't find any place that define the variable "span_loss‘’， so I ask this question, but they explain the concept.So I want to know someone can find the implement of
> the 'span_loss'.

I‘m trying to implement it.
